static/src/javascripts/projects/common/modules/commercial/contributions-utilities.js

```diff
@@ -196,8 +196,8 @@ const makeABTestVariant = (
         contributeURL = addTrackingCodesToUrl({
             base: contributionsBaseURL,
             componentType: parentTest.componentType,
-            componentId: campaignCode,
-            campaignCode,
+            componentId,
+            campaignCode: parentTest.campaignCode,
```

Comment:
Maybe we can extract `parentTest.campaignCode` into a `const` at the top of this function to avoid repeatedly retrieving the properly from the object?